modules/ml/src/main/java/org/apache/ignite/ml/naivebayes/discrete/DiscreteNaiveBayesTrainer.java

```diff
@@ -49,6 +52,9 @@
     /** The threshold to convert a feature to a discrete value. */
     private double[][] bucketThresholds;
 
+    /** Feature ids which should be skipped. By defaut all features are processed. */
+    private Collection<Integer> featureIdsToSkip = emptyList();
+
     /** {@inheritDoc} */
     @Override public <K, V> DiscreteNaiveBayesModel fit(DatasetBuilder<K, V> datasetBuilder,
                                                         Preprocessor<K, V> extractor) {
```

Comment:
@dehasi , How do you think, maybe we may remove featureIdsToSkip logic from trainers and models and in CompoundBayess we may create to Preprocessors that wrap input preprocessor by features filtering logic? In this case we avoid modification in Discrete and Gauss models and save this logic in one place.

Reply:
@avplatonov That's a good idea. I played with `map` function in an extractor,   I can do smth like `extractor = extractor.map(skipFeatures(featureIdsToSkip));` before call `fit`. 
But in this case, I make a copy of each vector.  Is it an acceptable approach?

Reply:
I think it won't be a big problem. In other case we can use proxy-object like vector-projection that wraps original vector and gives access only to non-filtered features

Reply:
ok, I used `skipFeatures(featureIdsToSkip)` and reverted Gaussian and Discrete Bayesses.